Component.vue

```diff
@@ -0,0 +1,29 @@
+<script setup>
+import { ref } from "vue";
+import axios from "axios";
+const pageTitle = "Data Fetching Example";
+const isLoading = ref(false);
+const data = ref([]);
+const fetchData = async () => {
+  try {
+    isLoading.value = true;
+    const response = await axios.get("https://api.example.com/data");
+    data = response.data;
```

Comment:
`data` here refers to a reference constant, hence the assignment should use the inner value `value`
```suggestion
    data.value = response.data;
```